lib/content/utils.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const _ = require('lodash');
+const isUUID = require('validator/lib/isUUID');
+
+
+const check = {
+
+  /**
+   * Determine if an id is in uuid format; update session for error
+   *
+   * @param  {object} config global config
+   * @param {object} req - HTTP Request
+   *
+   * @returns {boolean} false-not in uuid format / true-it is!
+   */
+  id: (config, req) => {
```

Comment:
Instead of passing `config` in to each of these, we should _use the global config_ that everything else is using